Packs/CortexResponseAndRemediation/Playbooks/playbook-Uncommon_execution_of_ODBCConf.yml

```diff
@@ -0,0 +1,715 @@
+        "position": {
+          "x": 450,
+          "y": 1700
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: deab0d59-be1c-454b-8043-540b7456529e
+    type: title
+    task:
+      id: deab0d59-be1c-454b-8043-540b7456529e
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1870
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: 8d7bf580-9887-46c9-85bc-05eab9fad48f
+    type: regular
+    task:
+      id: 8d7bf580-9887-46c9-85bc-05eab9fad48f
+      version: -1
+      name: Terminate Process Manually
+      description: |-
+        Dear Analyst,
+
+        During the remediation process, the playbook couldn’t terminate the process: ${alert.cgoname}
+
+        Please terminate the process manually if possible.
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 220,
+          "y": 1510
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 5b00bf39-f41c-4580-8ee8-a7eb6546221f
+    type: regular
+    task:
+      id: 5b00bf39-f41c-4580-8ee8-a7eb6546221f
+      version: -1
+      name: Close Alert - No results returned
+      description: Close the current alert as False Positive.
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      closeNotes:
+        simple: No indication of malicious activity was found. Closed automatically without any further action.
+      closeReason:
+        simple: Resolved - Handled by the playbook "Suspicious execution of ODBCConf"
+      id:
+        simple: ${alert.id}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1220,
+          "y": 1700
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 6bb015bc-9dc9-4669-8e89-17a7be7c0a70
+    type: title
+    task:
+      id: 6bb015bc-9dc9-4669-8e89-17a7be7c0a70
+      version: -1
+      name: No Results Found
```

Comment:
as we did find results just not the ones for remediation - please rename to "No malicious activity was found"